devnet-sdk/shell/env/devnet.go

```diff
@@ -60,10 +78,17 @@ func LoadDevnetFromURL(devnetURL string) (*DevnetEnv, error) {
 		return nil, fmt.Errorf("error fixing up devnet config: %w", err)
 	}
 
+	var ctrl surfaceGetter
+	ctrlFactory := schemeToBackend[parsedURL.Scheme].ctrlFactory
```

Comment:
Will this not panic if `parsedURL.Scheme` is not in the map? I am paranoid about any implicit type checks so I'd add an existence check here

Reply:
I added a comment. We indirectly ran that check above already

Reply:
Yeah that's what I mean by implicit type checks